scripts/preprocess.mjs

```diff
@@ -10,6 +10,7 @@ async function main() {
 	copyCSS();
 	generateRedirectFunctions();
 	await queryPluralKit();
+	await queryTeamInfo()
```

Comment:
The pedant in me would do `await Promise.all([queryPluralKit(), queryTeamInfo()]);` to run these two tasks in parallel, but... I don't think that matters much